### ISSUE TITLE
Form Form-Item prop 'error' is not passing input's error message

### ISSUE THREAD

**Issue by radoslawhryciow** [eleme/element-react#792]
### Description

I am trying to use 'error' prop of Form-Item to show custom validation message for one of my form inputs but form.state.error is still empty string.

### Solution

Pass error message from 'error' prop to input state.

### Additional Information

Information about 'error' prop was found on english page (https://eleme.github.io/element-react/#/en-US/form).

**Comment by thomask** [comment]
Yes, looks like this and show-message are not implemented (unless I am missing something)

**Comment by iddqdidkfa** [comment]
As [you can](https://github.com/ElemeFE/element-react/blob/master/src/form/FormItem.jsx) see `error` is a part of State of `Form.Item` component, not a property.
This behavior already changed in Vue version and even changed in documentation for React version, but not implemented, yet :)
